packages/hash/frontend/src/blocks/page/Comments/CommentThread.tsx

```diff
@@ -94,9 +97,11 @@ export const CommentThread: FunctionComponent<CommentThreadProps> = ({
         comment={comment}
         resolvable={
           authenticatedUser?.userAccountId ===
-            comment.metadata.provenance.createdById ||
+            extractOwnedByIdFromEntityId(comment.metadata.editionId.baseId) ||
           authenticatedUser?.userAccountId ===
-            comment.parent.metadata.provenance.createdById
+            extractOwnedByIdFromEntityId(
+              comment.parent.metadata.editionId.baseId,
+            )
         }
```

Comment:
Similar to the prior problem, I think this is a problem because the a comment cannot be resolved by the person who created it if they're responding to a comment they don't own, or commenting on a block they don't own.

Reply:
I found out that `ownedById` was not correct, please see [this slack thread](https://hashintel.slack.com/archives/C022217GAHF/p1669821735454679) (_internal_).